twitter-api/twitter-stream-map/config.py

```diff
@@ -0,0 +1,4 @@
+CONSUMER_KEY = 'jsjuaKAgWz4zDK2cKnGXf5Y30'
+CONSUMER_SECRET = 'vgAwEE77VIzVqvqVSS8yeSOMof3r38eF5vHiv6MhwQajk8sYHe'
```

Comment:
Recommendation: Remove these lines and change your secrets from apps.twitter.com :)

Reply:
already did but thx @enisimsar